environment_arm.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - bz2file
   - bzip2
   - cmake >=3.27,<3.28
-  - cspice =67
+  - cspice =67=*_11
```

Comment:
Do the changes in the main environment.yml also need to be applied to the ARM environment yamls?